packages/easy-coding-standard/build/target-repository/.github/workflows/fixer_test.yaml

```diff
@@ -0,0 +1,25 @@
+name: Fixer Test
+
+on:
+    pull_request: null
+    push:
+        branches:
+            - main
+
+jobs:
+    fixer_test:
+        runs-on: ubuntu-latest
+
+        steps:
+            -   uses: shivammathur/setup-php@v2
+                with:
+                    php-version: '7.3'
+                    coverage: none
+
+            -   run: composer require phpunit/phpunit
+
+            -   run: composer require symplify/easy-testing
+
+            -   run: composer require symplify/smart-file-system
+
+            -   run: vendor/bin/phpunit tests
```

Comment:
Where is the ECS? :thinking:

Reply:
It's not necessary - this is running in the ECS repo so it is there...

Reply:
But we need to test real example, where ECS is 3rd party dependency

Reply:
In that case let's add the checkout back. For the purpose of my test it's the same. In fact it's better since you use the ECS code from the commit instead of a possibly older version and there is no need for any sleep.